llama-index-integrations/llms/llama-index-llms-premai/llama_index/llms/premai/base.py

```diff
@@ -199,30 +199,25 @@ def chat(self, messages: Sequence[ChatMessage], **kwargs: Any) -> ChatResponse:
         if not response.choices:
             raise ChatPremError("ChatResponse must have at least one candidate")
 
-        chat_responses: Sequence[ChatResponse] = []
-
-        for choice in response.choices:
-            role = choice.message.role
-            if role is None:
-                raise ChatPremError(f"ChatResponse {choice} must have a role.")
-            content = choice.message.content or ""
-            chat_responses.append(
-                ChatResponse(
-                    message=ChatMessage(role=role, content=content),
-                    raw={
-                        "role": role,
-                        "content": content,
-                        "document_chunks": [
-                            chunk.to_dict() for chunk in response.document_chunks
-                        ],
-                    },
-                )
-            )
-
-        if "is_completion" in kwargs:
-            return chat_responses[0]
-
-        return chat_responses
+        choice = response.choices[0]
+        role = choice.message.role
+
+        if role is None:
+            raise ChatPremError(f"ChatResponse {choice} must have a role.")
+        content = choice.message.content or ""
+
+        chat_response = ChatResponse(
+            message=ChatMessage(role=role, content=content),
+            raw={
+                "role": role,
+                "content": content,
+                "document_chunks": [
+                    chunk.to_dict() for chunk in response.document_chunks
+                ],
+            },
+        )
+        print(chat_response)
```

Comment:
lets remove this debug print though

Reply:
Ahh forgot to remove this, let me do it rn